OpenMEEG/src/assemble.cpp

```diff
@@ -401,7 +411,11 @@ int main(int argc, char** argv)
 bool option(const int argc, char ** argv, const Strings& options, const Strings& files) {
     for ( auto s: options) {
         if (argv[1] == s) {
-            if (argc-2 < files.size()) {
+            unsigned minimum_nparms = files.size();
```

Comment:
are you testing this option somewhere?

Reply:
Yes. I made or to merge the two versions of H2ECoGM treating the index pf the surface as optional. This is what allowed me to merge the two.  There are other option with optional parameters that can now be documented in the option function. Previously, they were not because not providing one would have generated a wrong number of parameters error.

Reply:
Yes. I made or to merge the two versions of H2ECoGM treating the index pf the surface as optional. This is what allowed me to merge the two.  There are other option with optional parameters that can now be documented in the option function. Previously, they were not because not providing one would have generated a wrong number of parameters error.